api/src/main/proto/input/reqreply/chat/findOne.proto

```diff
@@ -32,3 +32,7 @@ message ChatFindOneResponse {
     }
   }
 }
+
+service ChatFindAllService {
+  rpc ChatFindAll(ChatFindOneRequest) returns (stream ChatFindOneResponse);
```

Comment:
that's what we discussed on yesterday's daily. do you really need a server-side stream here?
and why ChatFindOneRequest is used in the findAll endpoint?

Reply:
It's just example of stream usage. I know that this approach will produce many responses - that's sense. 
I've choosed FindAll to produce Flux<MongoChat>. 

Also FlindAll use FindOne because in order to simulate 1 by 1 emition of Objects and already written for that. I don't see much sense in writing same code with different name, but probably need advice